client/landing/jetpack-cloud/sections.js

```diff
@@ -0,0 +1,10 @@
+module.exports = [
+	{
+		name: 'jetpack-cloud',
+		paths: [ '/', '/scan', '/backups', '/settings' ],
+		module: 'landing/jetpack-cloud',
+		secondary: true,
+		group: 'jetpack-cloud',
+		enableLoggedOut: true,
```

Comment:
Are we really planning to handle this route as a logged out user? Seems to me that we should be using the login entry point for that, and customize it so it looks like the Jetpack Cloud. If that assumption is true, I don't see a good reason to enable this section for logged out users.

Reply:
To be honest I don't know why it was set to enableLoggedOut. It was inherited from the previous section.js file. I don't think it needs to change for this PR. Lets fix it in new one. When we figure out the login flow.

Reply:
I set it to `enableLoggedOut` sometime last week because I was under the impression after a review that we needed to do so, but if that's not the case then I'm sure it can be removed.